### PR TITLE
Add-EntraAdministrativeUnitMember

### PR DESCRIPTION
**Pull request by v-uansari**
Example tested and working.

### CODE REVIEW

**Comment by v-pughosh** [comment]
> Example tested and working.

Examples tested and working on dev machine

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraAdministrativeUnitMember.md [hunk 1]
```diff
@@ -33,14 +33,20 @@ Add-EntraAdministrativeUnitMember
...
 
 ## Examples
 
 ### Example 1: Add user as an administrative unit member
 
 ```powershell
-PS C:\>Add-EntraAdministrativeUnitMember -Id f306a126-cf2e-439d-b20f-95ce4bcb7ffa -RefObjectId d6873b36-81d6-4c5e-bec0-9e3ca2c86846
+Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All'
+$params = @{
+    Id = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraAdministrativeUnitMember.md:46]:
```suggestion
    ObjectId = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```

Comment (KenitoInc) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraAdministrativeUnitMember.md:46]: Replace `Id` with `ObjectId` in all instances

Comment (v-pughosh) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraAdministrativeUnitMember.md]: fixed and updated

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**